packages/sit-onyx/src/components/OnyxFlyout/OnyxFlyout.vue

```diff
@@ -194,6 +195,11 @@ watch([flyoutPosition, flyoutAlignment, flyoutWidth], async () => {
       width: max-content;
       font-family: var(--onyx-font-family);
 
+      &--disabled {
```

Comment:
I don't think we should change the content styles here...instead the flyout shouldn't open at all if its disabled